README.md

```diff
@@ -156,7 +156,7 @@ Run the linter, install, run tests
 Remember, you may need to set some environment variables to ensure your plugin
 is discovered and tested correctly.
 
-If you are modifying an existing plugin see `.travis.yml` for the required variables.
+If you are modifying an existing plugin see `.omero_plugin.yml` for the required variables.
```

Comment:
Did you mean for the file in this repo to be named "omero_plugin.yml" rather than "main.yml"? (Also, don't need the leading dot)